src/ui/sass/trumbowyg.scss

```diff
@@ -347,80 +347,88 @@ $slow-transition-duration: 300ms !default;
         }
     }
 
-    label {
-        display: block;
-        position: relative;
-        margin: 15px 12px;
-        height: 29px;
-        line-height: 29px;
-        overflow: hidden;
+    .trumbowyg-input {
+        &-row {
+            display: flex;
+            position: relative;
+            margin: 15px 12px;
+            border: 1px solid #DEDEDE;
+            overflow: hidden;
+        }
 
-        .trumbowyg-input-infos {
-            display: block;
+        &-infos {
             text-align: left;
-            height: 25px;
-            line-height: 25px;
             transition: all 150ms;
+            flex: 0 0 150px;
+            border-right: 1px solid #DEDEDE;
+            padding: 0 7px;
+            background-color: lighten($light-color, 5%);
 
-            span {
-                display: block;
+            label {
+                cursor: pointer;
```

Comment:
Why putting a cursor pointer on a label?

Reply:
this is a force of habit, it shows that the label is clickable, which really it is.
happy to remove it if not necessary

Reply:
I prefer to keep things as default :)